azurerm/resource_arm_kubernetes_cluster_other_test.go

```diff
@@ -90,6 +91,42 @@ func testAccAzureRMKubernetesCluster_basicVMSS(t *testing.T) {
 	})
 }
 
+func testAccAzureRMKubernetesCluster_privateLink(t *testing.T) {
+	resourceName := "azurerm_kubernetes_cluster.test"
+	ri := tf.AccRandTimeInt()
+	clientId := os.Getenv("ARM_CLIENT_ID")
+	clientSecret := os.Getenv("ARM_CLIENT_SECRET")
+	location := testLocation()
+
+	privateIpAddressCdir := "10.0.0.0/8"
+	privatefqdnRegex, err := regexp.Compile(".*.azmk8s.io")
```

Comment:
(same here) since this is only valid in Azure Public we'd be better to test this has a value set, rather than checking for a specific value here

Reply:
Resolved in new commit